regolith/dates.py

```diff
@@ -420,3 +420,28 @@ def is_between(thing, start=None, end=None):
     if is_after(thing, start) and is_before(thing, end):
         between = True
     return between
+
+def mdy(month, day, year, **kwargs):
+    """
+    Given a month, day, and year, returns a string with the date
+    in the following format: MM/DD/YY
+
+    Parameters
+    ----------
+    month: integer or string
+         the month.  if a string should be resolvable using regolith month_to_int
+
```

Comment:
remove empty lines.  If you want to see the format, please google for "numpy docstrings"

Reply:
this was deleted!